<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DigiFerro</class>
 <widget class="QMainWindow" name="DigiFerro">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1008</width>
    <height>700</height>
   </rect>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>DigiFerro</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="analyzeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Analyze New Images</string>
    </property>
   </widget>
   <widget class="QPushButton" name="fragmentButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Fragments</string>
    </property>
   </widget>
   <widget class="QPushButton" name="originalButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Original</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graphButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Distribution</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Image</string>
    </property>
   </widget>
   <widget class="QListWidget" name="imageList">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>141</width>
      <height>221</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="overallGraphButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Show Total Distribution</string>
    </property>
   </widget>
   <widget class="QLabel" name="fragmentLabel">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>10</y>
      <width>40</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>440</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Add Images</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="rectCBox">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Show box plots</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1008</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionUpload_file"/>
   </widget>
   <widget class="QMenu" name="menuHistory">
    <property name="title">
     <string>History</string>
    </property>
    <addaction name="actionUpload_file_2"/>
    <addaction name="actionShow_History"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuHistory"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionUpload_file">
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionUpload_file_2">
   <property name="text">
    <string>Open history file</string>
   </property>
  </action>
  <action name="actionShow_History">
   <property name="text">
    <string>Show History</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionShow_Graph">
   <property name="text">
    <string>DEMO: Show Graph</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
